### ISSUE TITLE
How to remove AlchemyCMS from a rails project?

### ISSUE THREAD

**Issue by hmswaffles** [AlchemyCMS/alchemy-guides#111]
Hello!

When I was first starting to develop my rails project, I installed AlchemyCMS, figuring it would come in handy later. Months elapsed, and in addition to not using it, now I'm finding that the incompatibility with Capistrano is a major drawback. What would you recommend I run for removing AlchemyCMS from a rails codebase?

Thanks,
Evan

**Comment by tvdeyen** [comment]
Hey.

Can you elaborate on what incompatibility with Capistrano you are having? It works perfectly with Capistrano for lots of users. I personally use Alchemy and Capistrano in dozens of apps. It's my go to deployment tool if I am not using Heroku or Docker. There is also a capistrano-alchemy gem that makes the integration even easier. 

Regardless of that, you can remove Alchemy by rolling back all its migrations and then simply remove the gem from the Gemfile. Remove alchemy folders from config and app/views and that's it.

**Comment by hmswaffles** [comment]
Sure.

For some reason, `cap production deploy` threw a huge hiccup when it got to the `rake assets:precompile` step. I tried this several times, and it resulted in a segmentation fault, and thousands of lines of stack-trace. Somewhere in the alchemy-cms docs (or maybe the alchemy-capistrano docs?), at the bottom of the page, it says that there is an incompatibility with Capistrano 3. 

Thanks for the info on removing alchemy -- I followed the steps, and now I'm able to run `cap production deploy.`
